raiden/network/matrixtransport.py

```diff
@@ -183,8 +183,9 @@ def start(
         self._join_discovery_room()
         self._discovery_room.add_listener(self._handle_discovery_membership_event, 'm.room.member')
 
+        gevent.spawn_later(1, self._inventory_rooms)
         gevent.spawn_later(2, self._ensure_room_peers)
-        gevent.spawn_later(5, self._send_queued_messages, queueids_to_queues)
+        gevent.spawn_later(3, self._send_queued_messages, queueids_to_queues)
```

Comment:
why is each greenlet spawned with an additional second in the future?

Reply:
No special reason, they are usually just to ensure these functions are run after the initial sync took place.